src/app/components/ErrorPage/index.jsx

```diff
@@ -27,6 +27,11 @@ const LongGridItemConstrainedMedium = styled(GridItemConstrainedMedium)`
   padding-bottom: 4rem;
 `;
 
+const ErrorTag = styled(Paragraph)`
+  padding-top: 40px;
+  margin-bottom: -60px;
```

Comment:
A minus margin is generally not preferred, is there any other way this can be done?